### DISCUSSION TITLE
CommandFluent(string cmdText, object parms = null)的防呆问题

### DISCUSSION THREAD

**Discussion by Rorschach331** [dotnetcore/FreeSql#987] [Q&A] [answered]
```
var cmdText = @"update table set col1 = 1 where col2 in @aa";
CommandFluent(string cmdText, new {aa})
```
上述代码在aa是一个string数组并且数组长度为0的时候会抛出
`No mapping exists from object type System.String[] to a known managed provider native type`
异常。
首先肯定是自己的业务逻辑问题，先做判断再执行即可避免。
但是如果执行`freesql.Ado.ExecuteNonQuery`方法就不会抛出异常，能否为CommandFluent加上防呆处理，参数数组没有元素不做更新即可？

**Comment by 2881099** [comment, marked as answer]
不同意，有时候让异常抛出有利于问题排查
